packages/flutter_tools/lib/src/macos/macos_ipad_device.dart

```diff
@@ -144,4 +144,7 @@ class MacOSDesignedForIPadDevices extends PollingDeviceDiscovery {
 
   @override
   Future<List<String>> getDiagnostics() async => const <String>[];
+
+  @override
+  List<String> get wellKnownIds => const <String>[];
```

Comment:
Maybe this ID should be put in a constant?
```suggestion
  List<String> get wellKnownIds => const <String>['designed-for-ipad'];
```